src/main/java/net/dv8tion/jda/core/entities/TextChannel.java

```diff
@@ -87,6 +88,12 @@
      */
     RestAction<Void> deleteMessagesByIds(Collection<String> messageIds);
 
+    RestAction<List<Webhook>> getWebhooks();
+
+    // RestAction<Webhook> createWebhook();
```

Comment:
Basically, I at first thought that webhooks should be created at the channel it was for.
But I didn't see how it would fit in so I moved it to `GuildController`.
This comment can be seen as my initial idea of where it logically should be but is currently not fitting in.